docs/reference/go-style-annotation-property-practice.md

```diff
@@ -0,0 +1,18 @@
+| Label                    | rdfs:label                                                                                                       | Y                             | Max 1 \*     | Full name of the term, must be unique.                                                                             | Free text                           | None                                                                                            | \* some ontologies have multiple labels for different languages, in which case, there should maximum be one label per language |
+| Definition               | IAO:0000115                                                                                                      | Y                             | Max 1        | A textual definition of ther term. In most ontologies, must be unique.                                             | Free text                           | database_cross_reference: reference materials used and contributors (in ORCID ID link format) | See [this document](https://douroucouli.wordpress.com/2019/07/08/ontotip-write-simple-concise-clear-operational-textual-definitions/) for guide on writing definitions                                                                     |
+| Contributor              | dcterms:contributor                                                                                                   | N (though highly reccomended) | No limit     | The ORCID ID of people who contributed to the creation of the term.                                                | ORCID ID (using full link)          | None                                                                                            |                                                                                                                                |
+| Synonyms                 | http://www.geneontology.org/formats/oboInOwl#has_exact_synonym, http://www.geneontology.org/formats/oboInOwl#has_broad_synonym, http://www.geneontology.org/formats/oboInOwl#has_narrow_synonym, http://www.geneontology.org/formats/oboInOwl#has_related_synonym                              | N                             | No limit     | Synonyms of the term.                                                                                              | Free text                           | database_cross_reference: reference material in which the synonymn is used                    | See [synonyms documentation](../reference/synonyms-obo.md) for guide on using synonyms                                                                     |
```

Comment:
Am I mistaken in thinking the synonyms should be hasExactSynonym rather than has_exact_synonym?

Reply:
hasExactSynonym is the short_form.  Unfortunately many ontologies also have a label attached - has_exact_synonym.  As this is not managed through imports, it's possible there may be some variations though.  Table should have IRI (or CURIE) and label columns I think.